habitat_baselines/rl/ddppo/algo/ddppo.py

```diff
@@ -61,7 +61,7 @@ def _get_advantages_distributed(
         self, rollouts: RolloutStorage
     ) -> torch.Tensor:
         advantages = (
-            rollouts.buffers["returns"][: rollouts.current_rollout_step_idx]
+            rollouts.buffers["returns"][: rollouts.current_rollout_step_idx]  # type: ignore
```

Comment:
@erikwijmans @mathfac Please be more specific on these errors, by doing `#type:ignore[error]`